src/util.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import kebabCase from 'lodash.kebabcase';
+
+export function getColorClassName( colorContextName, colorSlug ) {
+	if ( ! colorContextName || ! colorSlug ) {
+		return undefined;
+	}
+
+	return `has-${ kebabCase( colorSlug ) }-${ colorContextName }`;
+}
```

Comment:
This file as a whole isn't used.

Reply:
If we move back to a mix of `withColor` and `supports.color`, don't we need this file again then?

Reply:
Probably yeah.